### PR TITLE
Docs: added documentation about connecting Self-hosted Connectors with Redpanda Cloud cluster

### PR DESCRIPTION
**Pull request by bochenekmartin**
Task: https://github.com/redpanda-data/documentation/issues/1832#event-9991505525

### CODE REVIEW

**Review by tomasz-sadura — APPROVED** [review]

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 1]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
 4. Metrics are available at `localhost:9404/`.
 5. Use the Redpanda Console or Kafka Connect REST API to manage connectors.
 
+
+## Connecting to Redpanda Cloud cluster
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:195]:
```suggestion
## Connect to a Redpanda Cloud cluster
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 2]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+
+To connect to Redpanda Cloud cluster with Docker Compose:
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:197]:
```suggestion
To connect to a Redpanda Cloud cluster with Docker Compose:
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 3]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+1. Create Redpanda user, using RPK or Redpanda Console, `Security` tab.
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:198]:
```suggestion
1. Use `rpk` or Redpanda Console (**Security** tab) to create a Redpanda user.
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 4]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+2. Create ACLs for the user.
+3. Set username in `CONNECT_SASL_USERNAME` property.
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:200]:
```suggestion
3. Set the username in the `CONNECT_SASL_USERNAME` property.
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 5]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+4. Create a file containing user password, for example `passwords/redpanda-password/password` and set the path in `CONNECT_SASL_PASSWORD_FILE` property.
+5. Set `CONNECT_BOOTSTRAP_SERVERS` property. The value can be found in Redpanda Console, `Overview` tab, `Kafka API` section, `Bootstrap server URL` option.
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:202]:
```suggestion
5. Specify a value in the `CONNECT_BOOTSTRAP_SERVERS` property. You can view this value in Redpanda Console >**Overview**>**Kafka API**, in the `Bootstrap server URL` option.
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 6]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+6. Set `CONNECT_SASL_MECHANISM` property value to `"scram-sha-256"`.
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:203]:
```suggestion
6. Set the `CONNECT_SASL_MECHANISM` property value to `"scram-sha-256"`.
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 7]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+7. Set `CONNECT_TLS_ENABLED` property value to `"true"`.
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:204]:
```suggestion
7. Set the `CONNECT_TLS_ENABLED` property value to `"true"`.
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 8]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+    ├── ...
+    ├── passwords
+    │   └── redpanda-password
+    │       └──password                            # A file with SASL password
+    └── docker-compose.yaml                        # A docker-compose file
+
+
+### Redpanda Cloud Schema Registry
+
+For converters using Schema Registry (like AvroConverter, JsonConverter), use the following connector configuration properties to set up connection with Schema Registry:
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:244]:
```suggestion
For converters using Schema Registry (like AvroConverter, JsonConverter), use the following connector configuration properties to set up a connection with Schema Registry:
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 9]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+
+| Property                                        | Description                                                                                      |
+|-------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| `key.converter`                                 | Key converter class to use for the connector.                                                    |
+| `key.converter.schema.registry.url`             | Key converter Schema Registry URL, from the cluster `Overview` tab, `Schema Registry` section.   |
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:249]:
```suggestion
| `key.converter.schema.registry.url`             | Key converter Schema Registry URL, which you can view in the cluster **Overview**>**Schema Registry**.   |
```

File: docs/deploy/deployment-option/self-hosted/docker-image.mdx [hunk 10]
```diff
@@ -191,6 +191,80 @@ To connect with Docker Compose:
...
+| `key.converter.basic.auth.credentials.source`   | Key converter authentication method, should be `USER_INFO`.                                      |
+| `key.converter.basic.auth.user.info`            | Key converter user and password used for authentication, separated by a colon.                   |
+| `value.converter`                               | Value converter class to use for the connector.                                                  |
+| `value.converter.schema.registry.url`           | Value converter Schema Registry URL, from the cluster `Overview` tab, `Schema Registry` section. |
```
Comment (Feediver1) [inline docs/deploy/deployment-option/self-hosted/docker-image.mdx:253]:
```suggestion
| `value.converter.schema.registry.url`           | Value converter Schema Registry URL, which you can view in the cluster **Overview**>**Schema Registry**. |
```

**Comment by Feediver1** [comment]
Looks like you still need to add this new heading to the navtree @bochenekmartin 
That's why there is no preview for this new topic...all new headings must also be added to the navtree in sidebars.js.

**Comment by bochenekmartin** [comment]
@Feediver1 , the page is added to the navtree here: https://github.com/redpanda-data/documentation/blob/303fc19c1ea832a90dee3c53621e8bdf3b172d29/sidebars.js#L289

and the preview is available here: https://deploy-preview-1838--redpanda-documentation.netlify.app/docs/deploy/deployment-option/self-hosted/docker-image/#connect-to-a-redpanda-cloud-cluster

Is there any other code I need to update?

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**